lib/cookie.js

```diff
@@ -486,6 +505,8 @@ function parse(str, strict) {
 
   // ensure a default date for sorting:
   c.creation = new Date();
+  //NOTE: high-resolution time for the sorting in runtime
+  c.hrCreation = hrTimeNow();
```

Comment:
Naive outsider question but why not use an incrementing counter in module scope?